packages/shared/src/components/squads/PublicSquadEligibilityCard.tsx

```diff
@@ -0,0 +1,117 @@
+import { Button } from '../buttons/Button';
+import { ButtonSize, ButtonVariant } from '../buttons/common';
+import { Card } from '../cards/Card';
+import { Card as CardV1 } from '../cards/v1/Card';
+import { ClockPendingIcon, GlobeIcon, MiniCloseIcon } from '../icons';
+import { ProgressBar } from '../fields/ProgressBar';
+import { ActionType } from '../../graphql/actions';
+import { useLazyModal } from '../../hooks/useLazyModal';
+import { LazyModal } from '../modals/common/types';
+
+const MIN_POSTS = 3;
+
+interface Props {
+  postsCount: number;
+  inReview: boolean;
+}
+
+const PublicSquadEligibilityCard = ({
+  postsCount,
+  inReview,
+}: Props): ReactElement => {
+  const { shouldUseMobileFeedLayout } = useFeedLayout();
+  const { completeAction, checkHasCompleted, isActionsFetched } = useActions();
+  const { openModal } = useLazyModal();
+  const hideCard = useMemo(
+    () =>
+      isActionsFetched &&
+      checkHasCompleted(ActionType.HidePublicSquadEligibilityCard),
+    [checkHasCompleted, isActionsFetched],
+  );
+
+  const onDismiss = useCallback(() => {
+    completeAction(ActionType.HidePublicSquadEligibilityCard);
+  }, [completeAction]);
+
+  const onSubmit = useCallback(() => {
+    openModal({ type: LazyModal.PublicSquadSubmitForReview });
+  }, [openModal]);
+
+  if (hideCard) {
+    return null;
+  }
+
+  const isEligible = postsCount >= MIN_POSTS;
+  const effectivePostsCount = isEligible ? MIN_POSTS : postsCount;
+
+  const CardComponent = shouldUseMobileFeedLayout ? CardV1 : Card;
+
+  return (
+    <CardComponent
+      data-testid="publicSquadEligibilityCard"
+      className="justify-between !bg-transparent p-4"
+    >
+      <div className="flex items-center justify-between">
+        <div className="flex h-6 items-center gap-1 rounded-8 bg-surface-float pl-1 pr-2 text-text-tertiary typo-callout">
+          <GlobeIcon />
+          Public Squad
+        </div>
+        <Button
+          size={ButtonSize.Small}
+          variant={ButtonVariant.Tertiary}
+          icon={<MiniCloseIcon />}
+          onClick={onDismiss}
+          aria-label="Close squad eligibility"
+        />
+      </div>
+      <h3 className="font-bold typo-title3">
+        Get more traffic by making your Squad public
+      </h3>
+      <p className="text-text-tertiary typo-callout">
+        With Public Squads your posts will organically appear on the daily.dev
+        feed giving your content a lot more exposure
+      </p>
+      <div className="flex flex-col gap-1.5">
+        <div className="flex items-center justify-between typo-footnote">
+          <span className="text-text-secondary">Posts</span>
+          <span className="font-bold">{effectivePostsCount}/3</span>
+        </div>
+        <div className="h-2.5 rounded-10 bg-background-subtle">
+          <ProgressBar
+            percentage={Math.ceil((effectivePostsCount / 3) * 100)}
+            className="!static h-2.5 rounded-10 align-top"
```

Comment:
Why `!static`

Reply:
Because by default it's `absolute` :) And we definitely don't need absolute positioning here.

I didn't want to blow up the scope of this PR by refactoring the `ProgressBar` itself not to be absolute. But let me know if you think that's the right call here.

Reply:
No no all good